schemas/2015-12-01/Microsoft.Batch.json

```diff
@@ -7,6 +7,18 @@
     "batchAccounts": {
       "type": "object",
       "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[-\\w\\._]+$"
```

Comment:
This isn't your fault, but @xingwu1 I don't think this regex properly reflects our actual restriction, as it allows uppercase characters?